snapshotTestContent/Authoring-MVP3-(created-Aug-8)/src_snapshots/2.0.0-internal.3.1.0/snapshotVersion.json

```diff
@@ -0,0 +1 @@
+{"snapshotVersion":"2.0.0-internal.3.1.0"}
```

Comment:
"2.0.0-internal.3.1.0" should not be in src snapshots because its the current snapshot. Maybe you ran the update snapshot step twice?